### ISSUE TITLE
Cluster deletion does not clear the cluster config on rancher

### ISSUE THREAD

**Issue by dphe** [joyent/triton-kubernetes#135]
Cluster destroyed using triton-kubernetes does not remove the cluster config from the rancher dashboard.  Have to remove the config from rancher dashboard manually to again re-use the cluster name.

**Comment by fayazg** [comment]
That is correct. Terraform doesn't support removing environments yet. Looking for workarounds.
